src/components/chip-set/chip-set.tsx

```diff
@@ -273,38 +260,33 @@ export class ChipSet {
                         class={`mdc-text-field__input ${
                             hiddenInput ? 'hidden' : ''
                         }`}
-                        value={textValue}
+                        value={this.textValue}
                         onBlur={this.handleInputBlur}
                         onFocus={this.handleTextFieldFocus}
                         onInput={this.handleTextInput}
                         // Some browsers emit a change event on input elements, we need to stop
                         // that event from propagating since we are emitting our own change event
                         onChange={this.inputFieldOnChange}
+                        placeholder={this.searchLabel}
                     />
                 </div>
                 <label
                     class={{
                         'mdc-floating-label': true,
-                        'mdc-floating-label--float-above': !!(
-                            textValue || this.editMode
-                        ),
                         'mdc-text-field--disabled': this.disabled,
                         'mdc-text-field--required': this.required,
+                        'force-float': !!(this.value.length || this.editMode),
                     }}
                     htmlFor="my-text-field"
                 >
                     {this.label}
                 </label>
-                <label
-                    id="search-label"
-                    class="mdc-floating-label"
-                    htmlFor="my-text-field"
-                >
-                    {searchLabel}
-                </label>
                 <div class="mdc-line-ripple" />
-            </div>
-        );
+            </div>,
+            /*<div class="mdc-text-field-helper-line">
+                <div class="mdc-text-field-helper-text">{this.searchLabel}</div>
+            </div>*/
```

Comment:
I found this alternative to using a placeholder. It shows the text underneath the input field when the field is focused. The benefit is that the helper-text is visible even when the user has already started writing something. That's unless we cover it up with a menu of course, which we would in the case of the picker. Still an interesting approach which I think we should consider, which is why I temporarily left this here.